.github/workflows/pull-request.yml

```diff
@@ -6,41 +6,67 @@ on:
     branches:
```

Comment:
@matuella You may notice that most of the workflow is copy-pasted from `release` workflow. Github Actions has "composite actions" while let us reuse the workflow scripts, but it currently has a lot of limitations that makes unsustainable to use it as it is today

See more: https://github.com/actions/runner/issues/646

Reply:
Why don't we create a blocked issue that can eventually be solved when this runner issue has been solved?

Reply:
Also, why do we exactly need to setup the release credentials on a PR check? Why is fastlane called here?

Reply:
> Why don't we create a blocked issue that can eventually be solved when this runner issue has been solved?

Done: https://github.com/olmps/memo/issues/80

> Also, why do we exactly need to setup the release credentials on a PR check? Why is fastlane called here?

You're right, we shouldn't have secret api keys when building